src/__init__.py

```diff
@@ -20992,6 +21040,36 @@ def set_graphics_min_line_width(min_line_width):
         '''
         mupdf.fz_set_graphics_min_line_width(min_line_width)
 
+
+    @staticmethod
+    def get_word_delimiters():
+        """Return extra word delimiting characters."""
+        global g_word_delimiters
+        delims = [chr(c) for c in g_word_delimiters if c != 0]
+        return delims
```

Comment:
Is this right if g_word_delimiters is set to 10 characters, then reset to 5 characters? In this case i think it could be ....0.....0, where `.` is any non-zero character. I.e. we probably want to terminate at the first 0.

Also, don't need `global g_word_delimiters` - only required if we are modifying it.